pkg/utils/timeutil/duration.go

```diff
@@ -0,0 +1,68 @@
+package timeutil
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Duration is a wrapped time.Duration that supports durations as integers
+// or a ParseDuration string. If it's an integer it is interpreted in seconds instead
+// of being directly cast to time.Duration which would normally make it in nanoseconds.
+type Duration time.Duration
+
+// ErrInvalidDuration is returned when the duration can't be interpreted
+var ErrInvalidDuration = errors.New("the duration must be a string with time unit specified or an integer as seconds")
+
+// Get returns the underlying time.Duration type
+func (d Duration) Get() time.Duration {
```

Comment:
`Get` implies this is some kind of container for a duration, when it is actually just a duration.  Normally for conversion funcs I have seen them named `As*`.